packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -519,6 +526,8 @@ def get(self, track_id):
         user_data = request_args.get("user_data")
         user_signature = request_args.get("user_signature")
         nft_access_signature = request_args.get("nft_access_signature")
+        api_key = request_args.get("api_key")
```

Comment:
I'm wondering if there's any chance our URLs get too long. Probably not, but wondering if we should send this as a header instead at some point down the line. maybe when we evolve this system to use some kind of basic auth

Reply:
yeaa it's getting there